media/css/firefox/releasenotes.scss

```diff
@@ -417,3 +417,51 @@ $image-path: '/media/protocol/img';
         }
     }
 }
+
+.release-note {
+    .release-note-progressive-rollout-indicator {
+        @include text-body-md;
+        text-align: center;
+        width: 230px;
+        border: 2px solid $color-marketing-gray-20;
+        border-radius: $border-radius-md;
+        padding: $spacing-sm 0;
+        margin: auto;
+
+        img {
+            width: 100px
+        }
+
+        p {
+            margin-bottom: $spacing-sm;
+        }
+
+        details {
+            @include text-body-xs;
+            cursor: help;
+            padding: 0 $spacing-md;
+
+            summary {
+                position: unset;
```

Comment:
Is this needed? If it is, could it just be `static` instead of `unset`?

Reply:
I went with `unset` to use any inherited value if available, else the default, but just going straight to `static` works fine by the looks of it and is better cos it won't get overriden by inherited stuff 👍